wrapper/src/main/java/software/amazon/jdbc/util/RdsUtils.java

```diff
@@ -367,13 +382,27 @@ public boolean isGreenInstance(final String host) {
     return !StringUtils.isNullOrEmpty(preparedHost) && BG_GREEN_HOST_PATTERN.matcher(preparedHost).matches();
   }
 
+  public boolean isOldInstance(final String host) {
+    final String preparedHost = getPreparedHost(host);
+    return !StringUtils.isNullOrEmpty(preparedHost) && BG_OLD_HOST_PATTERN.matcher(preparedHost).matches();
+  }
+
```

Comment:
Calls to this method is always `!isOldInstance()`, you can use the IDE to invert the method and the calls for clarity
![image](https://github.com/user-attachments/assets/0cfc2f86-472b-43d3-9253-e5361ed223eb)